packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -59,4 +60,14 @@ describe('e2e api create', () => {
     await expect(Api.create({provider, timeout: 1})).rejects.toThrow(
       'Timed out in 1 ms.');
   });
+
+  it('Use custom metadata and test f it gets passed while creating API instance ', async () => {
+    const provider = 'wss://cennznet.unfrastructure.io/public/ws'; // Use Azalea
```

Comment:
I think there's some jest config for this

Reply:
I couldn't use the initApiPromise from jest/initApiPromise.. cause this needs customise ApiOptions (with metadata)..